src/main/java/com/github/dockerjava/api/model/SELContext.java

```diff
@@ -0,0 +1,44 @@
+package com.github.dockerjava.api.model;
+
+/**
+ * The context mode of bound volume in SELinux.
+ * Host path <code>shared</code> - can be used by all containers, <code>private</code> - by only this container
+ *
+ * @see http://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/
+ */
+public enum SELContext {
```

Comment:
I have big experience with jenkins where interfaces and enums causes limitations in future. When docker/selinux decides add new flag with abstract class user would be possible to implement it's own and pass it instead of waiting our releases.